src/output.ts

```diff
@@ -36,13 +37,20 @@ const tsconfig: Record<Runtime, any> = {
   [Runtime.node]: tsconfigBase,
 };
 
+function pathTo(module: string, binary: string) {
+  return require.resolve(join(module, 'bin', binary));
```

Comment:
It's not necessarily in `bin`, it just happens to be the in the case of these 2 modules.

Reply:
Agreed.  Removed to make it smoother.